### DISCUSSION TITLE
Importing a map into the DB editor

### DISCUSSION THREAD

**Discussion by SeanByrne08** [Spine-project/Spine-Toolbox#2381] [Support]
Hi!
I have been trying to use the importer to import in maps of different stochastic time series data, an example of how I am trying to import it is seen below. I am trying to import these maps to Object Classes and relationship Classes.
![image](https://github.com/spine-tools/SpineOpt.jl/assets/141043201/59ad42c8-dd79-4a0a-a639-1cb128cccd0e)

This is my current attempt:
![image](https://github.com/spine-tools/SpineOpt.jl/assets/141043201/bc043470-ba2c-4f21-85b7-cd955d3b48fa)

This is the imported result
![image](https://github.com/spine-tools/SpineOpt.jl/assets/141043201/f7e0a8b3-8ee3-42fd-b59b-19758ed2d730)

Any help on this problem would be greatly appreciated!
Thank you

**Comment by soininen** [comment]
There is at least two things you need to do here:
1. Check the **Compress Maps** check box in Importer specification editor. That will convert the leaf Maps to Time series.
2. Change the type of the time stamp column to datetime. This can be done from the 'A' button in Source data dock.

Also, your input data should probably be reshaped since it is missing the forecast column. Below is an example of how the data could look and how the Importer could be set up:

![image](https://github.com/spine-tools/Spine-Toolbox/assets/19147159/8cabd762-119f-4c23-9f1d-a3d3acb7f9aa)

**Reply by SeanByrne08** [reply]
Thank you very much for your response, the problem is now solved!
All the best,
Sean
